src/world/WorldServer.cpp

```diff
@@ -144,6 +144,11 @@ std::string readFileToString( const std::string& filename )
   // Close the file
   file.close();
 
+  // Remove all newlines from the file contents
+  fileContents.erase(std::remove(fileContents.begin(), fileContents.end(), '\n'), fileContents.end());
```

Comment:
we use whitespaces within parentheses
-> `fileContents.erase( std::remove( fileContents.begin(), fileContents.end(), '\n' ), fileContents.end() );`